tests/e2e/steps/steps_def.go

```diff
@@ -346,13 +346,13 @@ func (step *Step) validateCustomResourceStatus(res Resource, crNumStr string) er
 
 func (step *Step) validateDriverInstalled(res Resource, driverName string, crNumStr string) error {
 	crNum, _ := strconv.Atoi(crNumStr)
-	time.Sleep(60 * time.Second)
+	time.Sleep(20 * time.Second)
```

Comment:
Why is this change required for 1.6.1 and not 1.6.0?

Reply:
I don't believe this change is required for anything, it just reduces the amount of unnecessary delay in the testing substantially to accelerate the test runs.